generator/internal/rust/codec.go

```diff
@@ -408,6 +408,17 @@ func fieldFormatter(typez api.Typez) string {
 }
 
 func fieldSkipAttributes(f *api.Field) []string {
+	// oneofs have explicit presence, and default values should be serialized:
+	// https://protobuf.dev/programming-guides/field_presence/.
```

Comment:
Aside: the comment is fine for now.  Eventually we would want this to work for OpenAPI and discovery docs too.  I am not sure how those would represent `oneof`, something for all of us to keep in mind.

Reply:
Ack/